src/_gettsim/transfers/rente.py

```diff
@@ -606,16 +606,65 @@ def ges_rente_frauen_altersgrenze(
     return out
 
 
-def _ges_rente_arbeitsl_altersgrenze(
+@dates_active(end="1996-12-31", change_name="_ges_rente_arbeitsl_altersgrenze")
+def _ges_rente_arbeitsl_altersgrenze_ohne_vertrauenss(
     birthdate_decimal: float,
-    ges_rente_params: dict,
     ges_rente_regelaltersgrenze: float,
+    ges_rente_params: dict,
 ) -> float:
     """Calculate the age, at which an unemployed is eligible to claim the full
     pension (without deductions). This pension scheme allows for early retirement
     with deductions. Hence this threshold is needed as reference for calculating the
     zugangsfaktor.
 
+    Parameters
+    ----------
+    birthdate_decimal
+        See :func:`birthdate_decimal`.
+    ges_rente_regelaltersgrenze
+        See :func:`ges_rente_regelaltersgrenze`.
+    ges_rente_params
+        See params documentation
+        :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    lowest full retirement age for unemployed.
+
+    """
+    if (
+        birthdate_decimal
+        < ges_rente_params["first_birthyear_without_rente_für_arbeitsl"]
+    ):
+        out = piecewise_polynomial(
+            x=birthdate_decimal,
+            thresholds=ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][
+                "thresholds"
+            ],
+            rates=ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"]["rates"],
+            intercepts_at_lower_thresholds=ges_rente_params[
+                "altersgrenze_arbeitsl_abschlagsfrei"
+            ]["intercepts_at_lower_thresholds"],
+        )
+    else:
+        out = ges_rente_regelaltersgrenze
+
+    return out
+
+
+@dates_active(start="1997-01-01", change_name="_ges_rente_arbeitsl_altersgrenze")
```

Comment:
I do not understand the difference of this and the function `_ges_rente_arbeitsl_altersgrenze_vertrauenss` on line 715ff